dotcom-rendering/src/web/components/Callout/CalloutShare.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import { textSans } from '@guardian/source-foundations';
+import { Button, SvgShareCallout } from '@guardian/source-react-components';
+import { useState } from 'react';
+import { decidePalette } from '../../lib/decidePalette';
+
+const shareCalloutStyles = css`
+	display: flex;
+	align-items: center;
+`;
+const shareCalloutTextStyles = css`
+	display: inline-block;
+	${textSans.xsmall()}
+`;
+
+const shareCalloutLinkStyles = (format: ArticleFormat) =>
+	css`
+		color: ${decidePalette(format).text.richLink};
+		border-bottom: 1px solid ${decidePalette(format).text.richLink};
+		text-decoration: none;
+		font-weight: normal;
+	`;
+
+const placeholderCircle = css`
```

Comment:
Can we remove this placeholder now we have the icon in source?

Reply:
ah yes thanks - I think this stying is still needed I just need to rename it